kickstarts/centos8_build_machine.ks

```diff
@@ -1,4 +1,4 @@
-#version=RHEL8
+#version=RHEL9
```

Comment:
We don't use this file right now, but I'd like to keep it as a reference for setting up new build machines.